components/MashCards/ContactCard.tsx

```diff
@@ -2,20 +2,24 @@ import s from './MashCard.module.scss';
 import Link from 'next/link';
 import { MashReferral } from 'types';
 import { format } from 'date-fns';
+import MashTag from 'components/MashTags/MashTags';
 
 interface Props {
   mashReferral: MashReferral;
+  filter: string;
 }
 
-const ContactCard = ({ mashReferral }: Props): React.ReactElement => {
+const ContactCard = ({ mashReferral, filter }: Props): React.ReactElement => {
   return (
     <>
       <li className={s.row}>
         <div>
           <p className={`lbh-body-s govuk-!-margin-bottom-3 ${s.datetime}`}>
-            <span className="govuk-tag lbh-tag lbh-tag--green">
-              4 hours left
-            </span>{' '}
+            <MashTag
+              filter={filter}
+              createdAt={mashReferral.createdAt}
+              initialDecision={mashReferral.initialDecision}
+            />{' '}
```

Comment:
We should have margin here instead of empty space if we are using this for spacing (styling purposes)